scripts/apps/web-publisher/directives/SiteRoutesDirective.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * @ngdoc directive
+ * @module superdesk.apps.web_publisher
+ * @name sdSiteRoutes
+ * @requires publisher
+ * @description Directive to handle listing routes in web sites manager
+ */
+SiteRoutesDirective.$inject = ['publisher'];
+export function SiteRoutesDirective(publisher) {
+
+    class SiteRoutes {
+
+        constructor() {
+            this.scope = {site: '=site'};
+            this.templateUrl = 'scripts/apps/web-publisher/views/site-routes.html';
+        }
+
+        link(scope) {
+
+            /**
+             * @ngdoc event
+             * @name sdSiteRoutes#refreshRoutes
+             * @eventType broadcast on $scope
+             * @param {Object} data - subdomain for which to refresh routes
+             * @description event is thrown when modal window is closed and saved site is selected in WebPublisherManagerController
+             */
+            scope.$on('refreshRoutes', function(e, data) {
+                if (data === scope.site.subdomain) {
+                    queryItems();
+                }
+            });
+
+            function queryItems() {
```

Comment:
Since you've created an ES6 class (which is cool!) I think we can now take benefit of declaring these methods as part of the class, and even documenting them. A common pattern is naming it `_queryItems` and documenting it as `@private`.